src/utils/client.js

```diff
@@ -1,4 +1,4 @@
-import 'isomorphic-fetch';
+import 'unfetch/polyfill';
```

Comment:
Oh, okay, cool. Didn't realise that. Other ponyfills I've used (eg: `object-assign`) just expose methods you use instead of native. 

Or do you mean doing something like this?

```js
ourFetch = window.fetch() || unfetch();

ourFetch(...)
```

Anyway, if that's definitely the case with `unfetch` then happy to ponyfill, makes a lot more sense 👍